gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -26,6 +26,116 @@ syntax = "proto2";
 option java_package = "com.google.transit.realtime";
 package transit_realtime;
 
+message StopTime{
+  required int64 stop_sequence = 1;
+  required uint64 arrival_time = 2;
+  required uint64 departure_time = 3;
+  required string stop_id = 4;
+  optional TranslatedString stop_headsign = 5;
+  optional PickupType pickup_type = 6;
+  optional DropOffType drop_off_type = 7;
+  optional float shape_dist_traveled = 8;
+
+  enum PickupType{
+    REGULAR_PICKUP = 0;
+    NO_PICKUP = 1;
+    MUST_PHONE_AGENCY = 2;
+    MUST_ASK_DRIVER = 3;
+  }
+
+  enum DropOffType{
+    REGULAR_DROP_OFF = 0;
+    NO_DROP_OFF = 1;
+    MUST_PHONE_AGENCY = 2;
+    MUST_ASK_DRIVER = 3;
+  }
+
+}
+
+message StopTimeProperties{
+  optional string platform_id = 1;
+  optional TranslatedString stop_headsign = 2;
+  optional PickupType pickup_type = 3;
+  optional DropOffType drop_off_type = 4;
+  optional float shape_dist_traveled = 5;
+  optional string stop_id = 6;
+
+  enum PickupType{
+    REGULAR_PICKUP = 0;
+    NO_PICKUP = 1;
+    MUST_PHONE_AGENCY = 2;
+    MUST_ASK_DRIVER = 3;
+  }
+
+  enum DropOffType{
+    REGULAR_DROP_OFF = 0;
+    NO_DROP_OFF = 1;
+    MUST_PHONE_AGENCY = 2;
+    MUST_ASK_DRIVER = 3;
+  }
+}
+
+message VehicleProperties{
+  optional WheelchairAccessibleStatus wheelchair_accessible = 1;
+  optional BikesAllowedSatus biked_allowed = 2;
+
+  enum WheelchairAccessibleStatus{
+    UNKNOWN_WHEELCHAIR_ACCESSIBILITY = 0;
+    WHEELCHAIR_ACCESSIBLE = 1;
+    NOT_WHEELCHAIR_ACCESSIBLE = 2;
+  }
+
+  enum BikesAllowedSatus{
+    UNKNOWN_BIKES_ALLOWANCE = 0;
+    BIKES_ALLOWED = 1;
+    BIKES_NOT_ALLOWED = 2;
+  }
+}
+
+
+message Trip {
+
+  required string trip_id = 1;
+  optional string route_id = 2;
+  required TranslatedString trip_headsign = 3 ;
+  required string trip_short_name = 4;
+  optional int64 direction_id = 5;
```

Comment:
`TripDescriptor.direction_id` is `uint32`, so let's keep this consistent and change this to `uint32`.